### ISSUE TITLE
Intro to optimization in deep learning: Gradient Descent

### ISSUE THREAD

**Issue by sectooling** [sectooling/articles#18]
## TL;DR
This article gives a basic but detailed understanding of optimization strategy in neural networks with beautiful visualizations.
### Article Link
https://blog.paperspace.com/intro-to-optimization-in-deep-learning-gradient-descent/
### Author
Ayoosh Kathuria
## Key Takeaways
* Loss function
* high-level overview of local & global minima
* Gradient descent & its challenges
* Different types of gradient descent functions
